src/index.ts

```diff
@@ -1,4 +1,14 @@
-//Uncomment to see the choose pokemon page
-//const newTemplate = document.querySelector('#choose-page-template')!;
-//const app = document.querySelector('#app')!;
-//app.innerHTML = newTemplate.innerHTML
\ No newline at end of file
+import { activateStart } from './activateStart';
```

Comment:
it seems like you didn't commited/pushed `activateStart` file. Also, remember about extension (.ts) :)

Reply:
activateStart.ts file was added.

Should I add .ts extension when I import file? It works without extension but if it is not proper I will change it.

Reply:
my bad, it's ok :) sorry